src/themes/light/styles.js

```diff
@@ -268,7 +269,8 @@ export const styles = {
             "checked": {
                 "disabled": {
                     "boxShadow": "inset 0px 1px 5px 0px hsla(205, 50.3%, 30%, 0.051)",
-                    "backgroundColor": "hsla(210, 20%, 90.2%, 1.000)"
+                    "backgroundColor": "hsla(210, 20%, 90.2%, 1.000)",
+                    "cursor": "not-allowed",
```

Comment:
As the theme is generated by Sketch and this is something I can't represent for output without some heavier customisation to the plugin I wrote, could we manage the cursor type directly in the CSS / StyledComponent for now?